crates/services/shared-sequencer/src/config.rs

```diff
@@ -0,0 +1,37 @@
+use std::time::Duration;
+
+/// Endpoints for the shared sequencer client.
+#[derive(Debug, Clone)]
+pub struct Endpoints {
+    /// The RPC address of the sequencer chain tendermint API
+    /// (e.g. "http://127.0.0.1:26657")
+    pub tendermint_rpc_api: String,
+    /// The REST address of the sequencer chain tendermint API
```

Comment:
I think the comment isn't related to the arg documented. In my mind it the same as the one on clap defined here: https://github.com/FuelLabs/fuel-core/blob/38dfd569eb6f56446b3dedcc8bc4033ce228e333/bin/fuel-core/src/cli/run/shared_sequencer.rs#L15